src/error/errorHandler.ts

```diff
@@ -1,34 +1,41 @@
-import { Context } from '@azure/functions';
+import { InvocationContext } from '@azure/functions';
 
 import { Options } from '../middleware';
 import { stringify } from '../util/stringify';
 import { ApplicationError } from './ApplicationError';
 
 export const errorHandler = (
     error: unknown,
-    context: Context,
+    context: InvocationContext,
     opts?: Options,
-): {
-    [key: string]: unknown;
-} => {
-    if (error instanceof ApplicationError) {
-        context.log.error(`Received application error with message ${error.message}`);
-        return {
-            status: error.status,
-            body: error.body,
-        };
-    }
-
+    // TODO: Can we remove any?
```

Comment:
Bleibt das TODO?

Reply:
Ja, müssen wir uns später mal in den services anschauen.